dynamic-quorum/README.md

```diff
@@ -1,5 +1,60 @@
 # Dynamic Governance Quorum
 
-## Goal
-Update the governance quorum calculation to scale based on ratio of opposing votes.
+## Simple Summary
 
+A dynamic vote quorum that adjusts as a result of opposition, allowing uncontested proposals to pass at a minimum defined quorum, while requiring higher assurance on contested proposals.
+
+## Abstract
+
+This specification defines an upgrade to the `NounsDAOLogicV1` contract that replaces the static `proposal.quorumVotes` variable with a function that adjusts the vote quorum between a defined minimum and maximum as a result of voter opposition.
+
+## Technical Specification
+
+### Overview
+
+The dynamic quorum must be implemented as an upgrade to the Nouns DAO governance logic contract (`NounsDAOLogicV2`).
+
+- The following configuration values must be implemented as storage variables:
+  - Values:
+    - **Minimum Quorum Votes Basis Points** - The minimum basis point number of votes in support of a proposal required in order for a quorum to be reached and for a vote to succeed.
+    - **Maximum Quorum Votes Basis Points** - The maximum basis point number of votes in support of a proposal required in order for a quorum to be reached and for a vote to succeed.
+    - **Quorum Votes Offset Basis Points** - The offset used to modify the point at which the quorum adjustment activates.
+    - **Quorum Polynomial Coefficients (2)** - The coefficients used to control the dynamic quorum slope and curvature.
+  - Rules:
```

Comment:
@solimander should we also guard Offset Basis Points with an upper bound?

if we're capping max quorum at 6K, we should at least cap offset at the same value if not lower, perhaps at 4K?